alcs-frontend/src/app/features/application/applicant-info/application-details/parcel/parcel.component.html

```diff
@@ -69,10 +69,54 @@ <h4 [id]="parcel.uuid">Parcel #{{ parcelInd + 1 }}: Parcel and Owner Information
         <a (click)="openFile(parcel.certificateOfTitle)">{{ parcel.certificateOfTitle.fileName }}</a>
       </div>
     </div>
-    <div
-      class="full-width owner-information"
-      [ngClass]="{ 'crown-owner-information': parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN }"
-    >
+    <ng-container *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">
+      <h5 *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">Government Parcel Contact</h5>
```

Comment:
Do we need the *ngIf here if everything inside is already wrapped by <ng-container> with the same *ngIf condition?

This applies to all the other *ngIf, ie: Last Name, Ministry or Department, Phone, Email, etc.

Reply:
done, made changes